### DISCUSSION TITLE
Soil Volumetric Water Content Units/Standard Name

### DISCUSSION THREAD

**Discussion by Auguxst-char** [cf-convention/discuss#365] [Q&A about using CF] [answered]
I am working on putting some weather station data into CF-compliant NCDF format and was unsure what units or standard name (if it exists) should be used to describe soil %VWC data. Is there a recommended option or would:
>>
long_name = "soil_volumetric_water_content";
unitis = "%";
>>
be the best way to go about it?

Thanks

**Comment by JonathanGregory** [comment, marked as answer]
I think the appropriate standard name is volume_fraction_of_condensed_water_in_soil. "Condensed water" means liquid water and ice. It is dimensionless, with canonical units of 1, so you could put it in % if you prefer.

**Comment by efisher008** [comment]
Hi @Auguxst-char, @JonathanGregory,

Can we consider this issue as concluded? Were there any further questions before it is closed?

Best wishes,
Ellie
